tests/test_environments.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from bluepyentity.exceptions import BluepyEntityError
+from bluepyentity import environments as tested
+
+
+def test_create_forge__raises():
+
+    with pytest.raises(BluepyEntityError):
```

Comment:
Maybe, for future-proofing, we should add match="Unsupported 'store_overrides' keys"

Reply:
Added.